markdown/setup/tableau-desktop-driver-setup.md

```diff
@@ -0,0 +1,31 @@
+### Tableau Desktop
+[Link to product webpage](https://www.tableau.com/products/desktop).
+
+#### Adding the Amazon Timestream JDBC Driver
+1. [Download](https://github.com/awslabs/amazon-timestream-driver-jdbc/releases/latest) the Timestream JDBC driver shaded JAR file (e.g., `amazon-timestream-jdbc-2.0.0-shaded.jar`) and copy it to one of these
+   directories according to your operating system:
+   - **_Windows_**: `C:\Program Files\Tableau\Drivers`
+   ![Example](../images/tableau/tableau-driver-location.png)
+    - **_macOS_**: `~/Library/Tableau/Drivers`
+
+2. Using a text editor, copy and paste the entire <connection-customization> section of data source file. 
```

Comment:
Running Tableau with Connector should be done through the assembled taco file. While these instructions are good for development, they are not applicable to end users. Let's add a section on how to use Tableau with Generic JDBC driver (without using Connector).

Once the connector is ready to be published, we will need to add instructions on how to use Tableau with Connector.

Reply:
Removed steps 2 - 3 since they are not required